crates/hyperswitch_connectors/src/connectors/deutschebank/transformers.rs

```diff
@@ -227,11 +291,29 @@ impl
                         endpoint: item.data.request.get_complete_authorize_url()?,
                         method: common_utils::request::Method::Get,
                         form_fields: HashMap::from([
-                            ("reference".to_string(), reference),
-                            ("signed_on".to_string(), signed_on),
+                            ("reference".to_string(), reference.clone()),
+                            ("signed_on".to_string(), signed_on.clone()),
                         ]),
                     }),
-                    mandate_reference: None,
+                    mandate_reference: Some(MandateReference {
+                        connector_mandate_id: item.response.mandate_id,
+                        payment_method_id: None,
+                        mandate_metadata: Some(serde_json::json!(DeutschebankMandateMetadata {
+                            account_holder: item.data.get_billing_address()?.get_full_name()?,
```

Comment:
We should not be triggering 4xx from handle_response, either these fields needs to be checked while creating the payment, or rely completely on connector. But dont trigger a bad request after the request has been sent to the connector

Reply:
These fields are also being checked already while creating the payment